2_segmenting_data/2.segmenting_data.yml

```diff
@@ -3,6 +3,12 @@ channels:
   - conda-forge
 dependencies:
   - conda-forge::python=3.8.13
-  - conda-forge::jupyter=1.0.0
   - conda-forge::pandas=1.4.2
-  - conda-forge::cellpose=2.0.5
\ No newline at end of file
+  - conda-forge::cellpose=2.0.5
+  - conda-forge::scikit-image=0.19.2
+  - conda-forge::matplotlib=3.5.2
+  - conda-forge::jupyter=1.0.0
+  - conda-forge::jupyterlab
+  - conda-forge::ipykernel
+  - conda-forge::nb_conda_kernels
```

Comment:
Just a question, what are this dependency for? I haven't seen it used before.

Reply:
This is an important package for using `jupyter lab` in terminal. It enables me to explicitly set the conda kernel

https://github.com/Anaconda-Platform/nb_conda_kernels

> This extension enables a [Jupyter Notebook](http://jupyter.org/) or [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/) application in one [conda](https://conda.io/docs/) environment to access kernels for Python, R, and other languages found in other environments. When a kernel from an external environment is selected, the kernel conda environment is automatically activated before the kernel is launched. This allows you to utilize different versions of Python, R, and other languages from a single Jupyter installation.